pkg/util/feature/config_map_feature_provider.go

```diff
@@ -0,0 +1,103 @@
+package feature
+
+import (
+	"context"
+	"fmt"
+	"github.com/open-feature/go-sdk/openfeature"
+	"google.golang.org/appengine/log"
+	"strconv"
+)
+
+// ConfigMapFeatureProvider implements the OpenFeatureProvider interface
+type ConfigMapFeatureProvider struct {
```

Comment:
1. NewConfigMapFeatureProvider(kubeAPi Reader)
2. basic evalCOntext, flag name -> you can show
3. Fresh version feature flag value.
3b. configmap we will add help release.